MANUAL.html

```diff
@@ -10592,7 +10592,7 @@ <h3 id="object-lock-enabled-s3-bucket">Object-lock enabled S3 bucket</h3>
 </blockquote>
 <p>As mentioned in the <a href="#hashes">Hashes</a> section, small files that are not uploaded as multipart, use a different tag, causing the upload to fail. A simple solution is to set the <code>--s3-upload-cutoff 0</code> and force all the files to be uploaded as multipart.</p>
 <h3 id="standard-options-3">Standard options</h3>
-<p>Here are the Standard options specific to s3 (Amazon S3 Compliant Storage Providers including AWS, Alibaba, Ceph, China Mobile, Cloudflare, ArvanCloud, Digital Ocean, Dreamhost, Huawei OBS, IBM COS, IDrive e2, Lyve Cloud, Minio, Netease, RackCorp, Scaleway, SeaweedFS, StackPath, Storj, Tencent COS and Wasabi).</p>
+<p>Here are the Standard options specific to s3 (Amazon S3 Compliant Storage Providers including AWS, Alibaba, Ceph, China Mobile, Cloudflare, ArvanCloud, Digital Ocean, Dreamhost, Huawei OBS, IBM COS, IDrive e2, Lyve Cloud, Minio, Netease, RackCorp, Scaleway, SeaweedFS, StackPath, Storj, Tencent COS, Qiniu and Wasabi).</p>
```

Comment:
This file is auto generated - can you drop it from the commit please? rclone.1 and MANUAL.md